src/BrowseTables.jl

```diff
@@ -127,29 +127,36 @@ function writestyle(io::IO, path::AbstractString, inline::Bool)
     nothing
 end
 
-
+
 # high-level API
 
-function write_html_table(filename::AbstractString, table;
+function write_html_table(filename::AbstractString, tables;
                           title = "Table", caption = nothing,
                           options::TableOptions = TableOptions())
-    @argcheck Tables.istable(table) "The table should support the interface of Tables.jl."
+    ##
     open(filename, "w") do io
         write(io, HTMLHEADSTART)
         writetags(io -> writeescaped(io, title), io, "title")
         writestyle(io, options.css_path, options.css_inline)
         println(io, "</head>")    # close manually, opened in HTMLHEADSTART
         writetags(io, "body"; bropen = true) do io
+          for table in tables
+            @argcheck Tables.istable(table) "The table should support the interface of Tables.jl."
```

Comment:
I am not sure what purpose multiple tables per HTML file would serve. I think browser tabs are perfectly fine for having multiple tables open.